setup.py

```diff
@@ -1,55 +1,56 @@
+# Copyright (C) 2019 Canonical Ltd.
+# This file is part of ubuntu-advantage-client.  See LICENSE file for license.
+
 import glob
 import setuptools
-import subprocess
-import os
 
 from uaclient import config
 from uaclient.util import subp
 
 NAME = 'ubuntu-advantage-tools'
 
+raw_requirements, _err = subp(['./dev/read-dependencies'])
+INSTALL_REQUIRES = raw_requirements.rstrip('\n').split('\n')
+
+raw_test_requirements, _err = subp([
+    './dev/read-dependencies', '-r', 'test-requirements.txt']
```

Comment:
Thanks for this fixup. No sense in having two copies around. Will have to clean up read-dependencies at some point to gut py2 stuff. I'll look over your other branches and if you haven't already done this I'll add a low-weight issue.